submissions/ireneNakiyingi/database/transportationdb.sql

```diff
@@ -25,18 +23,17 @@ number_of_parents_children INT NOT NULL,
 number_of_siblings_spouses INT NOT NULL,
 point_of_embarkation VARCHAR(100) NOT NULL,
 PRIMARY KEY (id),
-FOREIGN KEY (id) REFERENCES registered_passengers(id)
+FOREIGN KEY (id) REFERENCES travel_id
 );
-select * from travel_details;
+
 
 -- table in the event of an accident, status = alive/dead
 CREATE TABLE flight_status(
 id INT NOT NULL AUTO_INCREMENT,
 traveller_id INT NOT NULL,
 status VARCHAR(10) NOT NULL,
 PRIMARY KEY (id),
-FOREIGN KEY (id) REFERENCES registered_passengers(id)
+FOREIGN KEY (id) REFERENCES traveller_id
```

Comment:
Use the comment above for this too.
FOREIGN KEY traveller_id REFERENCES blahblah blah

Reply:
I've made the changes. sorry, totally misunderstood the previous correction.